core/templates/dev/head/base.html

```diff
@@ -118,7 +118,7 @@
                 <div class="navbar-container">
                   <div class="navbar-header protractor-test-navbar-header pull-left">
                     <a ng-if="isWindowNarrow" ng-click="openSidebar()" class="navbar-brand oppia-navbar-menu oppia-transition-200">
-                      <img src="/images/general/menu.png">
+                      <i class="material-icons"style="color:#fff;margin-top:-5px">menu</i>
```

Comment:
1. Please pull out CSS styles into a new class.
2. Can you use ```<i class="material-icons">&#xE5D2;</i>``` instead? It seems more compatible,  based on the info [here](https://design.google.com/icons/#ic_menu). Ditto below.